### DISCUSSION TITLE
Will these work with ESP32 after flashing?

### DISCUSSION THREAD

**Discussion by AleXSR700** [pvvx/THB2#24] [Q&A]
Hi everyone,
I am not sure whether or not to buy a bunch of these or not. I do not use Zigbee or any hubs. I would like to connect these to a ESP32 and relay to wifi from there.

Does anybody know if this will work with the custom firmware?

Thanks for your help
Alex

P.S.:
If it will work, is there a recommendationas to whether to get the THB1, TH05 or TH05F?

**Comment by pvvx** [comment]
If the ESP32 supports [BTHome](https://bthome.io/) v2, then it will work.

**Reply by AleXSR700** [reply]
Any model recommendations? I could not find all info but what I gathered was that some models consume a lot of power. Not all are available but are there 2 models that are more energy efficient than the others?

**Comment by chrishome7** [comment]
@pvvx I have a basic question how do I change the screen from C° to F° Via the web GUI service's? have you thought about adding some of the basics and commands to the readme or implement something like the atc tellinkflasher that helps people less familiar with Hex/ble like myself?

**Reply by pvvx** [reply]
Your question is not clear.
The settings manager program is specified in the README.
https://pvvx.github.io/THB2/web/PHY62x2BTHome.html

**Reply by chrishome7** [reply]
Forgive my ignorance. I just want to change the display from showing degrees in Celsius (C° )  to Fahrenheit (F° )   I don't understand how exactly how to accomplish that using the settings manager.  What data or address do i send? or what command? I apologize im not too familiar with the subject yet.  TH05F_v20 Hardware 0019

**Reply by pvvx** [reply]
https://pvvx.github.io/THB2/web/PHY62x2BTHome.html
There are no HEX in the settings. Switching is done by checking the box.
![image](https://github.com/user-attachments/assets/49ef0e6d-d9eb-495f-b139-7c646519a4ad)

**Reply by chrishome7** [reply]
I apologize... that's exactly what I was looking for. For some reason it doesn't immediately show those settings in my browser when I initially connect, just the logs. I found it loads after I switch to one of the other tabs like OTA or Chart and then back to the config Tab. That's why I hadn't  seen it. This is great. Thank you for the help, and all the work you've done on this project and the Zigbee firmware.!
